cmd/prysmctl/withdrawal/cmd.go

```diff
@@ -0,0 +1,52 @@
+package withdrawal
+
+import (
+	"os"
+
+	"github.com/prysmaticlabs/prysm/v3/cmd"
+	"github.com/prysmaticlabs/prysm/v3/config/features"
+	"github.com/prysmaticlabs/prysm/v3/runtime/tos"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	BeaconHostFlag = &cli.StringFlag{
+		Name:  "beacon-node-host",
+		Usage: "host:port for beacon node to query",
+		Value: "http://localhost:3500",
+	}
+	FileFlag = &cli.StringFlag{
+		Name:  "file",
+		Usage: "file location for for the blsToExecutionAddress JSON or Yaml",
+	}
+)
+
+var Commands = []*cli.Command{
+	{
+		Name:    "set-withdrawal-address",
```

Comment:
```suggestion
		Name:    "set-withdrawal-addresses",
```
I don't feel very strongly about this, but since we publish an array of one or more objects, a plural might be more appropriate